processing/src/main/java/org/apache/druid/query/groupby/having/GreaterThanHavingSpec.java

```diff
@@ -121,12 +121,10 @@ public int hashCode()
   @Override
   public String toString()
   {
-    final StringBuilder sb = new StringBuilder();
-    sb.append("GreaterThanHavingSpec");
-    sb.append("{aggregationName='").append(aggregationName).append('\'');
-    sb.append(", value=").append(value);
-    sb.append('}');
-    return sb.toString();
+    return "GreaterThanHavingSpec{" +
+           "{aggregationName='" + aggregationName + '\'' +
```

Comment:
There's an extra `{` that got added here.

Reply:
Done